run-all.sh

```diff
@@ -19,8 +19,9 @@
 # Fail if anything fails.
 set -euv
 
-./run-test.sh libraryBuilds/com.google.code.gson-gson
-./run-test.sh libraryBuilds/org.joda-joda-convert
-./run-test.sh libraryBuilds/joda-time-joda-time
-./run-test.sh libraryBuilds/org.joda-joda-primitives
-./run-test.sh libraryBuilds/org.apache.commons-commons-lang3
+./run-test.sh com.google.code.gson-gson
+./run-test.sh org.joda-joda-convert
+./run-test.sh joda-time-joda-time
+./run-test.sh org.joda-joda-primitives
+./run-test.sh org.apache.commons-commons-lang3
+./run-test.sh com.fasterxml.jackson.core
```

Comment:
You need to sync to the upstream master to get the changes to run-test.sh. Make sure you've done that and it all passes first.